docs/ConvertDataOperation.md

```diff
@@ -0,0 +1,110 @@
+# Convert Data operation
+
+FHIR server implements $convert-data, a custom operation to enable data conversion from legacy formats to FHIR format. Currently it supports HL7v2 to FHIR conversion. It uses the Liquid templating engine and default conversion templates from the [FHIR Converter](https://github.com/microsoft/FHIR-Converter/tree/dotliquid) OSS project. 
+
+## How to use $convert-data
+
+Follow these steps to setup and use $convert-data API. Rest of this document details these steps. 
+
+1. Ensure that $convert-data is enabled on the FHIR server.
+1. Call $convert-data endpoint.
+1. Evaluate default templates.
+1. Customize templates if needed.
+1. Publish custom templates.
+1. Make templates accessible to the FHIR Server.
+1. Verify.
+
+### 1. Enable $convert-data
+You can enable $convert-data while deploying FHIR Server. In order to enable or disable the feature at a later stage, set the `FhirServer:Operations:ConvertData:Enabled` setting in the FHIR server to _true_.
+
+### 2. Make API call
+Make a call to ```<<FHIR service base URL>>/$convert-data``` with the [Parameters](http://hl7.org/fhir/parameters.html) resource in the request body as described below:
+
+| Parameter Name      | Description | Accepted values |
+| ----------- | ----------- | ----------- |
+| inputData      | Data to be converted | A valid JSON String|
```

Comment:
super nit: First two sentences in `Description` don't have `.` in the end, while two last sentences does.

Reply:
Fixed. Thanks!